### DISCUSSION TITLE
Disable tooltips and/or change color

### DISCUSSION THREAD

**Discussion by Slim08** [amnweb/yasb#80] [Q&A] [answered]
Is there a way to disable tooltips on hover from widgets like volume or clock? If not, then please change the color from bright white to something dark mode suitable.

**Comment by amnweb** [comment, marked as answer]
I will add the option to disable the tooltip per widget in the next release.

To style tooltip example

```
QToolTip {
    padding: 6px;
    color: #cdd6f4;
    font-size: 12px;
    background-color: #1e1e2e;
    border: 0px solid #313244;
    border-radius: 6px;
}
```
